sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -340,7 +340,7 @@ def add(self, message):
         :param message: The Message to be added to the batch.
         :type message: ~azure.servicebus.Message
         :rtype: None
-        :raises: :class:`ValueError`, when exceeding the size limit.
+        :raises: :class: ~azure.servicebus.exceptions.MessageContentTooLarge, when exceeding the size limit.
```

Comment:
If anything I'd make the ValueError subclassing here so the user can potentially avoid needing to import a specific exception type; save them a bit of cognitive overhead.

Reply:
I'd prefer MessageContentTooLarge because it's consistent with send(List)

Reply:
I agree with your point about consistency, I just wanted to also make users aware of what options they had when catching exceptions without needing to do other work/look at intellisense on other classes to figure out "oh I can just use a ValueError here like would be idiomatic".  (So if anything I'd make the same clarification both here and on send())

I won't bikeshed on this further, so if you feel strongly for how it is feel free to leave it and we can always add later on user feedback